modular_skyrat/modules/GAGS/greyscale_configs.dm

```diff
@@ -412,28 +412,135 @@
 	icon_file = 'modular_skyrat/modules/GAGS/icons/jumpsuit_teshari.dmi'
 
 // PANTS / SHORTS
+//Note, these icons are now in master_files per TG's sorting. There's no reason to NOT have these pieces in master files now that they're sorted out.
+//The rest will eventually join them - preferably with a cleaner method too instead of having to have unique configs for every single item's digi state.
+
+/datum/greyscale_config/buttondown_slacks_worn/digi
+	name = "Worn Buttondown with Slacks - Digitigrade"
+	icon_file = 'modular_skyrat/master_files/icons/mob/clothing/under/shorts_pants_shirts_digi.dmi'
```

Comment:
Something that I think we should start doing is to use some defines for those icon file paths, there's a lot of repetition and it means a lot of edits need to be made whenever we (you) decide to move them around. I think it might just simplify the workload over time, since defines auto-complete and are generally much shorter than those incredibly long file paths.

Reply:
Honestly I want to look into a way to automatically set digi greyscale configs, this always feels super copy-pastey to do; even on TG greyscale configs end up copy-paste, with the _worn version having duplicated .jsons, so its... all a mess

But yeah, I'll get some defines, SHORT_PANTS_SHIRTS_DIGI_ICONFILE or something; I'll go through and do the other ones in another PR though because this PR's already been worked on way too long